state.cpp

```diff
@@ -17,6 +17,9 @@
 #endif
 
 SystemState::SystemState() {
+	config.projectile_speed = 0;
+	config.turret_height = 0;
+	config.gravity = 0;
 #ifdef ARDUINO
```

Comment:
these should be initialized to the values in utility.h in the definition of the struct